bpdm-gate/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/repository/generic/BusinessPartnerRepository.kt

```diff
@@ -35,4 +35,10 @@ interface BusinessPartnerRepository : JpaRepository<BusinessPartner, Long>, Crud
     fun findByStageAndExternalIdIn(stage: StageType, externalId: Collection<String>, pageable: Pageable): Page<BusinessPartner>
 
     fun findByStage(stage: StageType, pageable: Pageable): Page<BusinessPartner>
+    fun findByStageAndBpnLInOrBpnSInOrBpnAIn(
```

Comment:
Does the logical 'or' take precedence over the 'and'? In any case I think its cleaner to write this in JPQL where you can set brackets how you need them.